spec/policies/admin_policy_spec.rb

```diff
@@ -15,30 +15,30 @@
 
   describe 'For a member that is a part of a company' do
     let(:members_company) { Company.find_by_company_number('5562728336')}
-    subject { described_class.new(admin).authorized? }
+    subject { described_class.new(members_company).authorized? }
```

Comment:
Not sure I follow this .... the intent of this test is to check auth for a member.  Shouldn't the statement then be:
```
subject { described_class.new(members).authorized? }
```
The way it is now, we're passing a `company` to the auth method (?)